pysoa/server/standalone.py

```diff
@@ -164,6 +165,20 @@ def django_main(server_getter):
     if args.settings:
         os.environ['DJANGO_SETTINGS_MODULE'] = args.settings
 
+    try:
+        # We have to import it manually, because we need to manipulate the settings before setup() is called, but we
+        # can't import django.conf.settings until after setup() is called.
+        django_settings = importlib.import_module(os.environ['DJANGO_SETTINGS_MODULE'])
+        django_settings.LOGGING = django_settings.SOA_SERVER_SETTINGS['logging']
+    except ImportError:
+        raise ValueError('Cannot import Django settings module `{}`.'.format(os.environ['DJANGO_SETTINGS_MODULE']))
```

Comment:
When `args.settings` is None, may this line throw another exception because `os.environ['DJANGO_SETTINGS_MODULE']` is undefined?

Reply:
No. `os.environ['DJANGO_SETTINGS_MODULE']` can never be undefined when we reach this point in code. This environment variable is either already defined _or_ `args.settings` is made a required parameter and the code errors before it gets here if the user doesn't provide that argument.